Ports/editline/package.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="editline"
+version="1.17.1"
+useconfigure="true"
+use_fresh_config_sub=true
```

Comment:
`true` here is also just a string; `'true'`

Reply:
Sorry I misread the commend. Should be good now :+1:

Reply:
```suggestion
use_fresh_config_sub='true'
```

Reply:
@emilytrau Still missing `use_fresh_config_sub='true'`...